Fix too many values to unpack error

moneypi opened the pull request:
<!--
Please make sure you've read and understood our contributing guidelines;
https://github.com/keras-team/keras/blob/master/CONTRIBUTING.md

Note:
We are no longer adding new features to multi-backend Keras (we only fix bugs), as we are refocusing development efforts on tf.keras. If you are still interested in submitting a feature pull request, please direct it to tf.keras in the TensorFlow repository instead.
-->

### Summary
Get error as:
'''
Using TensorFlow backend.
Traceback (most recent call last):
  File "cnn_seq2seq.py", line 68, in <module>
    input_text, target_text = line.split('\t')
ValueError: too many values to unpack (expected 2)
'''
### Related Issues

### PR Overview

- [N] This PR requires new unit tests [y/n] (make sure tests are included)
- [N] This PR requires to update the documentation [y/n] (make sure the docs are up-to-date)
- [Y] This PR is backwards compatible [y/n]
- [N] This PR changes the current API [y/n] (all API changes need to be approved by fchollet)

mikechen66 commented:
input_text, target_text, _  = line.split('\t')

Please have a look at the weblinks as follows.

github: https://github.com/keras-team/keras/pull/13863/commits/fd44e03a9d17c05aaecc620f8d88ef0fd385254b

moneypi commented:
> Solution:
> 
> change the original line of code
> input_text, target_text, = line.split('\t')
> to the following line of code
> input_text, target_text, _ = line.split('\t')
> 
> Please have a look at the weblink as follows.
> 
> github: [fd44e03](https://github.com/keras-team/keras/commit/fd44e03a9d17c05aaecc620f8d88ef0fd385254b)

Yeah, fd44e03 is just the PR you commenting ......
